src/main/java/com/api/Timesheets/services/UserServiceImpl.java

```diff
@@ -144,4 +145,30 @@ public void createUser(UserDTO userDTO){
         userDao.save(user);
 
     }
+
+    @Override
+    public void requestAccess(UserDTO userDTO) {
+        User user = User.fromUserDTO(userDTO);
+        user.setActive(false);
+        userDao.save(user);
+        String adminUserEmail = userDao.findByRole("ADMIN").getEmail();
```

Comment:
for now this is good, but going forward we can create a ENUM or globalconstants file to store these values like "ADMIN"